tinyagentos/app.py

```
@@ -662,50 +662,51 @@ async def _browser_reap_loop(app: FastAPI) -> None:
-            pass  # LiteLLM is optional
+                    await _litellm_migrate(data_dir)
+                except Exception:
+                    logger.exception("litellm prisma migration failed — virtual keys will not work")
+                resolved_secrets: dict[str, str] = {}
+                for backend in config.backends:
+                    name = backend.get("api_key_secret")
+                    if not name or name in resolved_secrets:
+                        continue
+                    try:
+                        rec = await secrets_store.get(name)
+                    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)